mllib/src/main/scala/org/apache/spark/mllib/clustering/KMeans.scala

```diff
@@ -193,20 +208,33 @@ class KMeans private (
 
     val initStartTime = System.nanoTime()
 
-    val centers = if (initializationMode == KMeans.RANDOM) {
-      initRandom(data)
-    } else {
-      initKMeansParallel(data)
-    }
+    // Only one run is allowed when initialModel is given
+    val numRuns = if (initialModel.nonEmpty){
+      if (runs >1 ) logWarning("Ignoring runs; one run is allowed when initialModel is given.")
```

Comment:
Please be careful about Scala style.  Look at [https://cwiki.apache.org/confluence/display/SPARK/Spark+Code+Style+Guide] or other parts of the codebase for examples.  We try hard to keep a consistent style.  Here:
```
val numRuns = if (initialModel.nonEmpty) {
  if (runs > 1) logWarning("Ignoring runs; one run is allowed when initialModel is given.")
  1
} else {
  runs
}
```